ssh/tailssh/tailssh_test.go

```diff
@@ -694,25 +693,6 @@ func TestSSHAuthFlow(t *testing.T) {
 					"accept": acceptRule.Action,
 				},
 			},
-			wantBanners: []string{"Welcome to Tailscale SSH!"},
-		},
-		{
-			name: "multi-check",
```

Comment:
Upstream changes in golang.org/x/crypto only support 2 authentication attempts, so the below no longer works. AFAIK, we don't currently use this many authentication steps in practice.